docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,84 @@
    "id": "840ea662",
```

Comment:
<p>"We can see in the results that the check calculates score on for each class in the dataset, and compare the score between our model and the simple model. " -&gt;</p><p>We can see in the results that the check calculates the score for each class in the dataset, and compares the scores between our model and the simple model.</p><p> </p><p>"the check also returns a value which contains all the information which is needed to be used for validation (using conditions)." -&gt;</p><p>"the check also returns a value which includes all of the information that is needed for defining the conditions for validation."</p><p> </p><p> </p><ul><li>scores - for each metric and class returns the numeric score</li><li>classes - the classes that exist in the data</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>